package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-redux-api",
-  "version": "5.1.0",
+  "version": "5.1.2",
```

Comment:
Should this be `5.1.1`?

Reply:
I think I was working from an old branch 😅 master is currently at 5.1.1.